### DISCUSSION TITLE
Welcome emails

### DISCUSSION THREAD

**Discussion by leewynne** [forem/forem#17994] [Feature Requests]
I have noticed that my welcome emails only go out to members that subscribed to newsletters. 

I think they should go out to all members, the only way I can think of implementing this is if this functionality were baked into forems in same way the verification email is sent out for people who create an account with email only.

**Comment by scrabill** [comment]
If I'm thinking about this the right way, there would have to be a place for members to specifically opt into receiving marketing emails at account creation.

**Reply by leewynne** [reply]
This already exists towards the end of the onboarding process.. but I am thinking if these boxes are unticked then that shouldn't really stop the member from receiving a welcome email that includes some really useful tips for getting started with the community, then no marketing news after that.

**Comment by michael-tharrington** [comment]
I agree with ya here, Lee!

But with a caveat… I think we should add some more details to the final section of onboarding where folks are opting in and out of email communication.

I feel like ideally we’d allow folks the ability to opt out from all email notifications if desired. Something like 👇:

![Screen Shot 2022-06-27 at 9 14 25 AM](https://user-images.githubusercontent.com/16007075/175953539-09586193-7856-42f5-adfe-367284c60f09.png)

I think we’re trying to keep this page simple for onboarding purposes and that’s why we don’t list the full range of notifications, but I think it’d also be good to mention on this page something like:

For more notification options, visit [your settings](https://dev.to/settings/notifications).

**Reply by michael-tharrington** [reply]
@leewynne I know we chatted about this on Slack, so I'm gonna move that convo here too.

Ya noted:

> Thanks for the response, yeah I agree about the onboarding being as simple as possible, as with the verification email, do you think a simple welcome email would be unwelcome if it was clear that it's a one-off and not a newsletter? I think so many people are missing out on that first welcome message with a load of community tips (even on Dev and Code newbies)

And I think that's a really good point! I responded with:

> If it’s a one-off (versus an onboarding stream of messages), I really can’t see folks getting too worked up about it. So in that case, it may be fine to just send it without them officially opting in. But, I’d like to hear others thoughts on it.
>
> If we were to add it to the list and make it opt in/opt out. I think it’d be ideal to add a specific option for it and check the box by default + maybe put in parentheses beside it “(one-time email)” or something to make it clear to folks what it is. Something like:
>
> "I want to receive a welcome email with information on how to get started (one-time email)"

**Reply by leewynne** [reply]
> "I want to receive a welcome email with information on how to get started (one-time email)"

I think this would do it for sure
